plugin/unmarshal/unmarshal.go

```diff
@@ -236,6 +236,7 @@ func (p *unmarshal) decodeVarint(varName string, typName string) {
 	p.P(`}`)
 	p.Out()
 	p.P(`}`)
+	p.P(`_ = `, varName) // satisfy ineffassign
```

Comment:
looking at the diff in generated code, this seems to change a lot more than necessary to fix the ineffassign check. is it possible to limit this codegen change to just the map type?